lib/charms/opensearch/v0/opensearch_distro.py

```diff
@@ -310,12 +310,15 @@ def write_file(self, path: str, data: str, override: bool = True):
             f.write(data)
 
     @staticmethod
-    def _run_cmd(command: str, args: str = None):
+    def _run_cmd(command: str, args: str = None, stdin: str = None) -> str:
```

Comment:
It is not very clear why do we need "stdin" on top of the args

Reply:
Improved the docstring, check it out